README.md

```diff
@@ -81,7 +81,7 @@ shipped with deployinator
 - Run the tailer as a background service (using whatever init flavor you like)
 
 ```sh
-    ./bin/deployinator-tailer.rb &
+    ./deployinator/deployinator-tailer.rb &
```

Comment:
Merged. After running the binstub command, did you still need to use this pull request?